src/clients/archwayd/index.js

```diff
@@ -60,10 +77,31 @@ class DefaultArchwayClient {
     debug(command, ...parsedArgs);
     return spawn(command, parsedArgs, { ...options, encoding: 'utf8' });
   }
+
+  async runJson(subCommand, args = [], { printStdout = true, ...options } = {}) {
+    const archwayd = this.run(
+      subCommand,
+      [...args, '--output', 'json'],
+      { stdio: 'pipe', maxBuffer: 1024 * 1024, ...options }
+    );
+
+    printStdout && archwayd.stdout.pipe(process.stdout);
+
+    try {
+      const { stdout } = await archwayd;
+      const lines = stdout.replace(/\r/g, '').split('\n');
+      const jsonLines = lines.filter(line => line.startsWith('{'));
+      const jsonOutput = jsonLines.pop() || '{}';
+      return JSON.parse(jsonOutput);
+    } catch (e) {
+      throw new ArchwayRunError(e.stderr);
+    }
+  }
 }
 
-class DockerArchwayClient extends DefaultArchwayClient {
-  constructor({ archwaydVersion = DefaultArchwaydVersion, testnet, ...options }) {
+// TODO: deprecate Docker support
```

Comment:
this we could add together with the clean up after docker support removal issue

Reply:
Answered above